rogue/src/main.rs

```diff
@@ -368,7 +368,6 @@ impl State {
         dissipates.run_now(&self.ecs);
         let mut spawns = EntitySpawnSystem{};
         spawns.run_now(&self.ecs);
-        process_entity_spawn_request_buffer(&mut self.ecs);
```

Comment:
Removing these solves the discordance with entities spawning before animations complete. It turns out this is *all* we needed to do, since we clear the spawn buffer in `run_cleanup_systems`.